massa-ws/src/lib.rs

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2022 MASSA LABS <info@massa.net>
+//! Unsigned WebSocket management
+#![warn(missing_docs)]
+#![warn(unused_crate_dependencies)]
+#![feature(bound_map)]
+
+use jsonrpsee::{core::error::SubscriptionClosed, SubscriptionSink};
+use serde::Serialize;
+use tokio::sync::broadcast::Sender;
+use tokio_stream::wrappers::BroadcastStream;
+
+/// Re-exports libraries to not require any additional
+/// dependencies to be explicitly added on the module(s) side.
+#[doc(hidden)]
+pub mod __reexports {
```

Comment:
Never saw this way of re-exporting. The idiomatic way (I believe) is to `pub use` the part you want to re-export. I have made this in my previous PR : https://github.com/massalabs/massa/blob/testnet_18/massa-execution-exports/src/lib.rs#L56 (I re-export some massa-sc-runtime stuff)

Reply:
IMHO, this way is used to share an entire dependency. But in our case may be a `pub use` could be used.

Reply:
I don't think is necessary to share a whole module yes.

Reply:
done